src/shared.js

```diff
@@ -2056,16 +2056,6 @@ export const getCollectionsByVisit = async (data) => {
     return collections;
 };
 
-export const getCollectionById = async (data, collectionId) => {
-
-    const response = await getParticipantCollections(data.token);
-    let collectionExists = false;
-    if(response.code != 404) {
-        collectionExists = response.data.some(col => col.id === collectionId)
-        };
-    
-    return collections;
-};
```

Comment:
Is this being deleted because the `getCollectionById`function  is not being referenced or used?